index.js

```diff
@@ -215,16 +217,23 @@ function gatherMetaData(url, body) {
       image = image.replace(RE_AMPS, '&')
     }
 
-    var description
-    cheerioBody.find('meta[property="og:description"]').each(function (i, elem) {
-      var elemContent = cheerio(elem).attr('content')
-      description = elemContent && elemContent.trim()
-    })
+    var getSimpleValue = function(cheerioTagQuery, targetAttribute) {
+      var foundValue
+
+      cheerioBody.find(cheerioTagQuery).each(function (i, elem) {
+        var elemContent = cheerio(elem).attr(targetAttribute)
+        foundValue = elemContent && elemContent.trim()
+      })
+
+      return foundValue
+    }
 
     return {
       title: (title && title.trim()) || 'Untitled',
       image: image,
-      description: description
+      description: getSimpleValue('meta[property="og:description"]', 'content'),
+      amphtml: getSimpleValue('link[rel="amphtml"]', 'href'),
+      canonical: getSimpleValue('link[rel="canonical"]', 'href')
```

Comment:
Interesting. Didn't know about this. Is it in the `meta` property or just in the page somewhere?